edb/schema/indexes.py

```diff
@@ -185,6 +206,57 @@ def get_displayname_static(cls, name: sn.Name) -> str:
         shortname = cls.get_shortname_static(name)
         return shortname.name
 
+    def get_all_kwargs(
+        self,
+        schema: s_schema.Schema,
+    ) -> s_expr.ExpressionDict:
+        kwargs = s_expr.ExpressionDict()
+        all_kw = self.__class__.get_field('kwargs').merge_fn(
```

Comment:
Does just `self.get_field` not work? (Or does that get resolved to something else?)
I'd probably go with `type(self)` over `self.__class__`